app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -116,6 +116,7 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         }
     }
 
+    @SuppressWarnings("LongMethod")
```

Comment:
Suppressed this warning as I stepped over the 75 lines limit, but I don't think it should be part of this PR.

If you'd like this to be refactored I suggest to open a new issue.